statsmodels/robust/scale.py

```diff
@@ -220,10 +220,17 @@ def __init__(self, d=2.5, tol=1e-08, maxiter=30):
         self.maxiter = maxiter
 
     def __call__(self, df_resid, nobs, resid):
-        h = (df_resid)/nobs*(self.d**2 + (1-self.d**2)*\
+        h = df_resid * 1. /nobs*(self.d**2 + (1-self.d**2)*\
```

Comment:
whitespace around operator, try to avoid backslash.  +1 for removing unnecessary parens